storybook.native/story-loader.js

```diff
@@ -0,0 +1,45 @@
+// @flow
```

Comment:
Somehow, this should have been gitignored. Can you inquire why this was not the case? Thank you!